cmd/main.go

```diff
@@ -13,32 +13,41 @@ func main() {
 	inputBuffer := compiler.NewInputBuffer()
 	scanner := bufio.NewScanner(os.Stdin)
 	for {
+		printPrompt()
 		scanner.Scan()
 		command := scanner.Text()
 		inputBuffer.Buffer = command
 
-		if strings.HasPrefix(inputBuffer.Buffer, "."){
-			switch compiler.DoMetaCommand(inputBuffer){
+		if strings.HasPrefix(inputBuffer.Buffer, ".") {
+			switch compiler.DoMetaCommand(inputBuffer) {
 			case compiler.MetaCommandSuccess:
-				continue;
+				continue
 			case compiler.MetaCommandUnrecognizedCommand:
-				fmt.Printf("Unrecognized command %s \n", inputBuffer.Buffer)
-				continue;
-			}}
-
+				fmt.Printf("Unrecognized command %q \n", inputBuffer.Buffer)
+				continue
+			}
+		}
 
 		var statement compiler.Statement
-			switch compiler.PrepareStatement(inputBuffer, &statement){
-			case compiler.PrepareSuccess:
-				
-			case compiler.PrepareUnrecognizedStatement:
-				fmt.Printf("Unrecognized command at start of %s \n", inputBuffer.Buffer)
-				continue;
+		switch compiler.PrepareStatement(inputBuffer, &statement) {
+		case compiler.PrepareSuccess:
+
+		case compiler.PrepareUnrecognizedStatement:
+			fmt.Printf("Unrecognized command at start of %q \n", inputBuffer.Buffer)
+			continue
+
+		case compiler.PrepareSyntaxError:
+			fmt.Println("Syntax error. Could not parse statement.")
+			continue
 		}
 
 		compiler.ExecuteStatement(statement)
 		fmt.Println("Executed")
-			
-		}
-		
+
 	}
+
+}
+
+func printPrompt() {
+	fmt.Printf("db > ")
+}
```

Comment:
Can we use `PrintPrompt` instead? Let's stick with this format for function names and `camelCase` for variable names 👌